app/services/users.js

```diff
@@ -2,9 +2,39 @@ const { User } = require('../models');
 const errors = require('../errors');
 const bcrypt = require('bcryptjs');
 const { SALTROUNDS } = require('../constants');
-
+const logger = require('../logger');
 exports.createUser = user =>
   bcrypt.hash(user.password, SALTROUNDS).then(hash => {
     user.password = hash;
     return User.create(user).catch(error => Promise.reject(errors.databaseError(error.message)));
   });
+
+exports.signInUser = ({ email, password }) => {
```

Comment:
this method should be here. we dont put bussiness logic in services. 
here you should only put a method to do the findOne in the db